modules/nw-metallb-bgpadvertisement-cr.adoc

```diff
@@ -70,5 +70,5 @@ This BGP attribute applies to BGP sessions within the Autonomous System.
 
 |`spec.peers`
 |`string`
-|Optional: Peers limits the BGP peer to advertise the IPs of the selected pools to. When empty, the load balancer IP is announced to all the BGP peers configured.
+|Optional: Specifies the `metadata.name` value for each `BGPPeer` resource to inform about the IP addresses from your selected IP address pool. By default, the load balancer IP is announced to all configured `BGPPeer` resources. Use this field to limit the announcement to specific BGP peers.
```

Comment:
Could we rephrase it to say the field is to specify all the BGP peers to which IP assigned to LB service from the IP address pool is advertised. if the list is not specified than the IP address is advertised to all the bgp peers configured. With BGP use of 'advertised'  is relevant and 'announcement' is relevant to selected nodes. The selected nodes with nodeSelector, IP is service is announced. With BGP based on selected nodes the IP of LB service or host prefix is advertised to BGP peers. Selective if .spec.peers has a list, all if this field is missing.

Reply:
Thanks Arti. I've had another go at this description. Please see the update and let me know if this works better.